chartfx-math/src/main/java/de/gsi/math/MathBaseFast.java

```diff
@@ -69,17 +69,17 @@ public static float sinDeg(float a) {
     }
 
     private static void init() {
-        modulusRad = (int) (2.0 * Math.PI * precision);
+        modulusRad = (int) (2.0 * java.lang.Math.PI * precision);
         modulusRadQuater = modulusRad >> 2;
         modulusDeg = 360 * precision;
         sinRadLookup = new float[modulusRad]; // sine lookup table [rad]
         sinDegLockup = new float[modulusDeg]; // sine lookup table [deg]
 
         for (int i = 0; i < modulusRad; i++) {
-            sinRadLookup[i] = (float) Math.sin((double) i / precision);
+            sinRadLookup[i] = (float) java.lang.Math.sin((double) i / precision);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.sin' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)